docs/source/inference.mdx

```diff
@@ -112,6 +112,23 @@ from optimum.intel import OVModelForCausalLM
 model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=True)
 ```
 
+> **NOTE:** `load_in_8bit` is enabled by default for the models larger than 1 billion parameters.
+
+There are also alternative compression options for a different performance-accuracy trade-off:
+
+| Option                                 | Description                        |
+|--------------------------------------|--------------------------------------|
+| `fp16`                               | Float16 weights.   |
+| `int8`                               | INT8 weights.      |
+| `int4_sym_g128`                      | INT4 *symmetrically* quantized weights of most of the Linear layers with group size 128. The rest of the Linear layers and all Embeddings are quantized to INT8.   |
```

Comment:
would keep this explanation outside the table :
```suggestion
| `int4_sym_g128`                      | *symmetric* INT4  with group size 128 |
```
and add the second sentence after the table (+ could be nice to extend a bit on "most of the Linear layers")

Reply:
addressed

Reply:
thanks!